app/styles/application.scss

```diff
@@ -14,14 +14,23 @@ body > .ember-view
 
   .el-label{@include label; color: $color-text-main;}
   .el-label-dark{ @include label; color: $color-text-dark;}
-
+/*
+$fs6 : 2.3rem;
+$fs5 : 1.5rem;
+$fs4 : 1.3rem;
+$fs3 : 1.1rem;
+$fs2 : 1rem;
+$fs1 : 0.8rem;
+$fs0 : 0.6rem;
```

Comment:
Not really happy with `fsX` names. I have no idea what it means :stuck_out_tongue_winking_eye: